db/migrate/20150414034414_add_profile_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddProfileToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :profile, :jsonb, null: false, default: "{}"
+    add_index  :users, :profile, using: :gin
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.